### ISSUE TITLE
Enabling sACN viewer causes sACN to output improperly to pathscape gateways.

### ISSUE THREAD

**Issue by onerandomusername** [stagehacks/Cue-View#278]
**Describe the bug**
sACN output on the network is corrupted or frozen when Cue View is watching sACN

**To Reproduce**
1. Enable sACN out
2. Try to view sACN out in Cue View

**Expected behavior**
Network works fine and is not disrupted

**Actual Behaviour**
Network has corrupted or frozen sACN, pathways gateway freezes.

**Screenshots**

**Desktop (please complete the following information):**
- Eos Console:
- Surface Pro 7, Windows 11 pro, Eos 3.2.9 using Surface Dock 2 to  a tplink 8 port switch, with second laptop connected to said switch.

**Additional context**
Disconnecting ethernet from my cue view laptop causes sACN to restore. Closing CueView causes sACN to restore. Unsure of root cause. More details after the show, though sending now in case for whatever reason recreating it is hard.

**Comment by sparks-alec** [comment]
I cannot replicate on my test rig here- a MacBook running Nomad, another Mac running Cue View, and a DMXKing node.

Pathway does have a unique take on sACN (including security features) so it's possible it's a vendor-specific or console-specific issue.

I will get myself onto a network with a Pathport node tomorrow.

**Comment by onerandomusername** [comment]
I can dump my pathport config in a little while. We have another show tonight and it's our final one before I tear down the rig. I'll be able to try and recreate at home, though it'll be hard to configure. 
It's a really odd issue, and I'm disappointed I can't use Cue View for this reason. 

(Given said bug, I don't feel comfortable running the Eos cue view either, as I do have a few other bugs during setup, but none that occur during operation, though I would love to use cue view)

**Comment by onerandomusername** [comment]
Here's what i believe is the necessary information, don't believe I can get more right now
![image](https://github.com/user-attachments/assets/06c183f1-a1aa-4fa2-afe7-af07d0f4a0bc)

**Comment by sparks-alec** [comment]
Was there a DHCP server on your network? The Eos console, a router, or something?

**Comment by onerandomusername** [comment]
So, I noticed I'm on v1.1.0 so I'm attempting to download 1.2.1 but its taking some time to download. Will update shortly.

**Comment by onerandomusername** [comment]
> Was there a DHCP server on your network? The Eos console, a router, or something?

Yes, for DHCP I'm using a cheap Gl-inet travel router, as all it does is serve DHCP and act as a false gateway) 

My configuration for DHCP is 10.101.67.1 for the DHCP server, and assigned devices get 10.101.67.xxx,
but the two laptops and gateways all have static IPs.

Nomad Console: 10.101.1.101
Cue View Laptop: 10.101.1.103

Pathport Gateway: 10.101.80.20

**Comment by onerandomusername** [comment]
At first launch, Cue View doesn't even work with sACN even though the device can retrieve it 
![image](https://github.com/user-attachments/assets/eb45c058-b981-4fc4-a3a3-f072a5584ac2)

**Comment by onerandomusername** [comment]
Aight, this might mean nothing, but using 0.0.0.0 as the IP address made it finally *possibly* not output bad data to the lighting

**Comment by onerandomusername** [comment]
Issue reared its head again as I was shutting down and took sACN control the moment the console went down. 

Apologies for the live updates rather than putting it all in one comment. I'm tearing down the rig today and don't know when it'll be back up, or able to be troubleshooted, hence my trying to document everything before strike.

**Comment by sparks-alec** [comment]
No problem! Really appreciate you sharing while you had the rig together! I'm sorry your Cue View experience was frustrating, I'll keep looking into it.

I believe the sACN Cue View plugin totally ignores that supplied IP address (since sACN uses a designated Multicast group) and doesn't ever send anything. So the problems you experienced are very mysterious!

Am I correct in my understanding that _Cue View_ took sACN control? What was the indication of that? Lights changing, an indicator in Pathscape, or such?

**Comment by onerandomusername** [comment]
> Am I correct in my understanding that _Cue View_ took sACN control? What was the indication of that? Lights changing, an indicator in Pathscape, or such?

I was unable to diagnose this much, as I didn't have a third computer (lol)
Watching Pathport, it would show me that DMX would "freeze" output and lock to the same values that Cue View received when it connected. Oddly, Cue View continues to update with the sACN from the console.

**Comment by sparks-alec** [comment]
Did you have any lights attached to the output of the gateway? How did they respond when Cue View was launched?

**Comment by onerandomusername** [comment]
> Did you have any DMX lights attached to the output of the gateway? How did they respond when Cue View was launched? Flickering, blackouts, etc

Output of DMX was an oldddddd dimmer rack. No idea of model. They would hold their state, which to my knowledge, is what they're supposed to do when sACN output is lost. I will be testing with a simple light directly to the DMX run as another form of diagnostic. I can also look into using a device to receive the stream of DMX (though I currently do not possess one of those tools)

> Can you confirm you were running Pathscape on the same computer as Cue View?

For troubleshooting, I ran both Pathscape on the console (as I was running ETCnomad) AND on the same computer as Cue View, and had the same issue. 

I will try to set up a minimal network (Two computers, switch, pathways gateway. No router, no DHCP, just those.) I have some basic leds I can use as status indicators.

**Comment by Duderda** [comment]
I can confirm this issue.
I came across it twice, which led me to block CueView's access to my sACN-network.

I'm using EOS with Swisson Gateways. No DHCP server on the network.

The issue occurs once the search-function on cueview is triggered and the sACN-Stream is detected.

The consequence was lagging DMX-output. Fades where not taking place, but instead snapping to the target values seconds later. 

The weird thing ist, that even shutting down CueView or the PC it's running on did not resolve the issue. I basically had to restart the whole rig and network-setup.

I love the search function since it's just incredibly comfortable. 
But while this issue persists, it would be nice to have a filter for it.

Or a module-toggle for the different protocols.
sACN-viewer is a killer for me atm.
